src/lib/datasource/abstract.coffee

```diff
@@ -16,8 +20,35 @@ class GridDatasourceAbstract
     return this
 
   ###
+   * Get/Set number of results returned from a query
+   * @param  {number} limit
+   * @returns {this|number}
+  ###
+  limit: ( limit ) ->
+    return @_limit if typeof limit is 'undefined'
```

Comment:
I don't agree. If there is a wrong case, it should throw an error. In this case I want to preserve the plain getter/Setter aspect